src/test/java/org/jabref/logic/git/GitPreferencesTest.java

```diff
@@ -0,0 +1,50 @@
+package org.jabref.logic.git;
+
+import javafx.beans.property.BooleanProperty;
+
+import org.jabref.logic.preferences.AutoPushMode;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+class GitPreferencesTest {
+
+    private GitPreferences gitPreferences;
+
+    @BeforeEach
+    void setUp() {
+        gitPreferences = new GitPreferences(true, AutoPushMode.ON_SAVE, "", "");
+    }
+
+    @Test
+    void constructorInitialisesValues() {
+        assertThat(gitPreferences.getAutoPushEnabled()).isTrue();
```

Comment:
JabRef more uses plain JUnit5.

Please use `assertTrue` instead of (........isTrue())